catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/sierra/subjects/SierraConceptSubjects.scala

```diff
@@ -51,7 +48,15 @@ trait SierraSubjects extends MarcUtils with SierraConcepts {
   //      Note that only concepts from subfield $a are identified; everything
   //      else is unidentified.
   //
-  private def getSubjectsForMarcTag(bibData: SierraBibData, marcTag: String) = {
+  def getSubjectswithAbstractConcepts(
+    bibData: SierraBibData): List[Subject[MaybeDisplayable[AbstractConcept]]] =
+    getSubjectsForMarcTag(bibData, "650") ++ getSubjectsForMarcTag(
+      bibData,
+      "648") ++ getSubjectsForMarcTag(bibData, "651")
```

Comment:
nit: the auto formatting has mangled this, making it look like the old `getSubjects` would make it easier to read.